### DISCUSSION TITLE
can fetch foreign tables?

### DISCUSSION THREAD

**Discussion by joaquinns** [supabase/supabase#5897] [Questions] [answered]
I create a function in javascript to fetch the data like this

```
export const getVideos = async () => {
  const { data: videos, error } = await supabase.from('videos').select(`
    *, user_id (*)
  `)
  return [error, videos]
}
```

but when i'm trying to fetch the data of the foreign table in this case user_id show undefined for example: videos.user_id.username

**Comment by GaryAustin1** [comment]
Your syntax looks correct.
If RLS is blocking the table user_id is pointing to you will get null for that part of the return.

**Comment by steve-chavez** [comment]
Can you show the full error message?

**Comment by joaquinns** [comment, marked as answer]
i solve the problem reseting the dev server
